Data/models/Bookmark.swift

```diff
@@ -141,6 +141,17 @@ public final class Bookmark: NSManagedObject, WebsitePresentable, Syncable, CRUD
         return getFoldersInternal(bookmark: nil)
     }
     
+    public class var hasFavorites: Bool {
+        guard let count = count(predicate: NSPredicate(format: "isFavorite == true")) else { return false }
```

Comment:
i see `NSPredicate(format: "isFavorite == true")` copied into three places inside this class, may just make a private prop or function that returns this. 🤷‍♀